.github/workflows/prerelease.yml

```diff
@@ -9,7 +9,7 @@ on:
 
 env:
   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-  GO_VERSION: '1.18'
+  GO_VERSION: '1.19'
```

Comment:
As far as I can see, this env var is only used by `setup-go` action. Could you confirm it?

If so, wouldn't be better to change these actions to use `go-version-file` instead of `go-version`? That will make that we would not have to update it.

An example: https://github.com/newrelic/newrelic-prometheus-configurator/blob/main/.github/workflows/_push_image.yaml#L34

Reply:
Yes, I tried to update it but I was getting hit by this error in the pipeline while running Windows tests https://github.com/newrelic/nri-kafka/actions/runs/4183067220/jobs/7247031898 🤔. I'm going to double-check it then, to get rid of the `GO_VERSION`.

Reply:
We've been hit by this [pipeline error](https://github.com/newrelic/nri-kafka/actions/runs/4183855614/jobs/7248747546) when moving from`GO_VERSION` to `go-version-file` on Windows.  I tried to find the error in a reasonable time but I couldn't find it; as we have defined a [future task](https://issues.newrelic.com/browse/NR-83905) to give some love to the pipeline, we can address it there.

Reply:
LGTM :)